File: Go/Slice/all-elements-after-index-in-slice.go
package main

import (
	"fmt"
)

func main() {
	// Create a slice.
	slice := []string{"Apple", "Orange", "Plum", "Cherry", "Banana"}
	// Get the elements after the index.
	fmt.Println(allElementsAfterIndexInSlice(slice, 2))
}

// Get all the elements of the slice after the given index and return them.
func allElementsAfterIndexInSlice(slice []string, index int) []string {
	return slice[index:]
}

File: Dart/Slice/all_elements_after_index_in_slice.dart
import "dart:core";

void main() {
  // Create a new list of strings.
  final List<String> randomSlice = <String>["Bob", "Alice", "Joe"];
  // Get all the element after the given index.
  print(getAllElementsAfter(randomSlice, 0));
}

// Get all the element after a given index in a slice.
List<dynamic> getAllElementsAfter(final List<dynamic> slice, final int index) {
  return slice.sublist(index + 1);
}
